src/sofa/pbrpc/rpc_server.cc

```diff
@@ -79,6 +79,16 @@ bool RpcServer::IsListening()
     return _impl->IsListening();
 }
 
+bool RpcServer::RegisterWebServlet(const std::string& path, Servlet servlet)
```

Comment:
to make interface easier to use, it is better to add "bool owns_servlet" param, if is true, then the RpcServer would take ownership of deleting the servlet.